src/main/java/edu/iis/powp/features/CommandsFeature.java

```diff
@@ -8,7 +8,7 @@
 public class CommandsFeature {
 
 	private static PlotterCommandManager commandManager;
-	public static ArrayList<String> commandList = new ArrayList<>();
+	//public static ArrayList<String> commandList = new ArrayList<>();

```

Comment:
FIX TO MERGE: Remove comments! (Ideally file should be returned to original state.)